hs_core/templates/search/search.html

```diff
@@ -32,47 +45,49 @@ <h4 class="panel-title">
-                                                style="background-color:rgb(158, 0, 96)"
-                                            {% elif item.1 > 32 and item.1 <= 57 %}
-                                                style="background-color:rgb(142, 0, 112)"
-                                            {% elif item.1 > 57 and item.1 <= 100 %}
-                                                style="background-color:rgb(127, 0, 127)"
-                                            {% elif item.1 > 100 and item.1 <= 178 %}
-                                                style="background-color:rgb(111, 0, 143)"
-                                            {% elif item.1 > 178 and item.1 <= 317 %}
-                                                style="background-color:rgb(95, 0, 159)"
-                                            {% elif item.1 > 317 and item.1 <= 563 %}
-                                                style="background-color:rgb(79, 0, 175)"
-                                            {% elif item.1 > 563 and item.1 <= 1000 %}
-                                                style="background-color:rgb(63, 0, 191)"
-                                            {% elif item.1 > 1000 and item.1 <= 1779 %}
-                                                style="background-color:rgb(47, 0, 207)"
-                                            {% elif item.1 > 1779 and item.1 <= 3163 %}
-                                                style="background-color:rgb(31, 0, 223)"
-                                            {% elif item.1 > 3163 and item.1 <= 5624 %}
-                                                style="background-color:rgb(15, 0, 239)"
-                                            {% elif item.1 > 5624 and item.1 <= 10000 %}
-                                                style="background-color:rgb(0, 0, 254)"
-                                            {% endif %}>{{ item.1 }}
-                                        </span>
-                                        {#<a href="{{ request.get_full_path }}&amp;selected_facets={{ key }}_exact:{{ item.0|urlencode }}">{{ item.0 }}</a>#}
-                                    </li>
+                                    {% if item.1 >= 2 %}
+                                        <li class="list-group-item">
+                                            <input class="faceted-selections" id="{{ key }}-{{ item.0 }}" value="{{ key }},{{ item.0 }}" type="checkbox" class="pull-left"> &nbsp;&nbsp; {{ item.0 }}
+                                            <span class="label label-default label-pill pull-right badge"
+                                                {# {% if item.1 <= 1 %} #}
+                                                {# style="background-color:rgb(254, 0, 0)"  #}
+                                                {# {% elif item.1 > 1 and item.1 <= 2 %} #}
+                                                {#  style="background-color:rgb(238, 0, 16)" #}
```

Comment:
Thanks for the suggestions! I will take a look at it later. 

> On Dec 10, 2015, at 16:32, Hong Yi <notifications@github.com> wrote:
> 
> In hs_core/templates/search/search.html:
> 
> > -                                            {% elif item.1 > 3163 and item.1 <= 5624 %}
> > -                                                style="background-color:rgb(15, 0, 239)"
> > -                                            {% elif item.1 > 5624 and item.1 <= 10000 %}
> > -                                                style="background-color:rgb(0, 0, 254)"
> > -                                            {% endif %}>{{ item.1 }}
> > -                                        </span>
> > -                                        {#<a href="{{ request.get_full_path }}&amp;selected_facets={{ key }}_exact:{{ item.0|urlencode }}">{{ item.0 }}</a>#}
> > -                                    </li>
> > +                                    {% if item.1 >= 2 %}
> > +                                        <li class="list-group-item">
> > +                                            <input class="faceted-selections" id="{{ key }}-{{ item.0 }}" value="{{ key }},{{ item.0 }}" type="checkbox" class="pull-left"> &nbsp;&nbsp; {{ item.0 }}
> > +                                            <span class="label label-default label-pill pull-right badge"
> > +                                                {# {% if item.1 <= 1 %} #}
> > +                                                {# style="background-color:rgb(254, 0, 0)"  #}
> > +                                                {# {% elif item.1 > 1 and item.1 <= 2 %} #}
> > +                                                {#  style="background-color:rgb(238, 0, 16)" #}
> @xuezhaokun got it, thanks for the clarification. I think the background colormap selection needs improvements, use of bright red color to represent small number of faceted results is against convention in my opinion. You can check out http://colorbrewer2.org/ to select a nice colormap scheme to represent number of faceted results, but this can be addressed after AGU. For now, I'll give +1 and merge it.
> 
> —
> Reply to this email directly or view it on GitHub.
>